### PR TITLE
[FLINK-6786] [metrics] Deduplicate QueryScopeIntoTest

### PR DESCRIPTION
**Pull request by zentol**

### CODE REVIEW

File: flink-runtime/src/test/java/org/apache/flink/runtime/metrics/dump/QueryScopeInfoTest.java [hunk 1]
```diff
@@ -20,58 +20,140 @@
 
 import org.junit.Test;
 
-import static org.apache.flink.runtime.metrics.dump.QueryScopeInfo.INFO_CATEGORY_JM;
```
Comment (pnowojski) [inline flink-runtime/src/test/java/org/apache/flink/runtime/metrics/dump/QueryScopeInfoTest.java]: I would leave that static imports for readability

**Comment by zentol** [comment]
PR is now removing the duplicated test file, thanks for noticing.

File: flink-runtime/src/test/java/org/apache/flink/runtime/metrics/dump/QueryScopeInfoTest.java (hunk 1 above)
Comment (zentol) [inline flink-runtime/src/test/java/org/apache/flink/runtime/metrics/dump/QueryScopeInfoTest.java]: Whether to use static imports or not is currently rather undefined in Flink.

In this case i would remove the static import to make it very clear in the test that we're comparing to a constant that is defined outside the test.

Comment (pnowojski) [inline flink-runtime/src/test/java/org/apache/flink/runtime/metrics/dump/QueryScopeInfoTest.java]: ok

**Comment by zentol** [comment]
merging.

**Pull request closed without merging**